test/View/RouteNotFoundStrategyTest.php

```diff
@@ -16,9 +16,12 @@
 use Zend\Mvc\MvcEvent;
 use Zend\Mvc\View\Http\RouteNotFoundStrategy;
 use Zend\View\Model\ViewModel;
+use ZendTest\Mvc\EventManagerIntrospectionTrait;
 
 class RouteNotFoundStrategyTest extends TestCase
 {
+    use EventManagerIntrospectionTrait;
```

Comment:
As mentioned above, you are copying the helper methods into every test case, instead of just having them available in the test suite.

Since they are pure helper methods, you can just have them as static methods in a separate class.